news/2022-03-07-community-meetings-recap.md

```diff
@@ -0,0 +1,54 @@
+
+<iframe width="90%" height="315" src="https://www.youtube.com/embed/ut_SOreClcc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Recently, the format of these meetings has changed slightly in an effort to be both more concise and targeted (as they tended to drag on and become a bit sprawling in terms of the topics covered), and also to get more people involved in hosting them. 
+
+[DigitalHypno](https://osu.ppy.sh/users/4384207) has stepped as the first of hopefully many hosts for these new shortform meetings, which [started back in February in response to outcry regarding a temporary change made to the score multipliers in lazer](https://www.youtube.com/watch?v=d66pU5lsHvE).
+
+If you missed that meeting, here's a (very) quick rundown of what was talked about:
+
+- The change made to mod multipliers a few weeks ago and picked up by concerned commentators on social media was a temporary fix to address problems with the lazer-exclusive playlist system.
+  - In lazer, some mods can be adjusted, such as setting the rate at which DoubleTime speeds up the track (in stable, this is a flat 1.5x speed always).
+  - Players could set adjusted DoubleTime at 1.01x speed to gain the flat 1.12x score multiplier, giving them an unfair advantage over those who didn't if freemod was enabled in the playlist/lobby.
+  - Temporarily disabling mod multipliers was intended as a stopgap to fix the above issue while a more robust solution was added.
+- **There are no plans to actually remove mod multipliers in the future.**
+- Playlist hosts will have full freedom over mod multipliers to tailor this kind of play to their individual wishes (tournaments, spotlights-type contests, etc). Global leaderboards will use some kind of 'default' setting.
+- In the future, score multipliers won't be "baked in" on scores but instead exist as seperate components of the score, meaning that mods can be freely rebalanced and scores appropriately recalculated on the fly to match.
+
+The most recent meeting was hosted yesterday, focused on the topic of leaderboards in the future. Check it out below:
+
+<iframe width="90%" height="315" src="https://www.youtube.com/embed/HimCHAnPCCY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+If you're just interested in a quick summary of what was discussed:
+
+- Improvements to the way scores are saved and stored on the game's end now allow for a lot more flexibility in deciding how to use and show them.
+- Scores set now or previously on lazer won't add to global pp, since they aren't properly verified (no anticheat), but most other stats will still apply.
+- Almost all of the 'processors' needed to handle lazer scores are complete, with the exception of play time and medal awarding.
```

Comment:
is there some kinda logic to this that i'm not aware of? both types of quotes are used for the same purpose

```suggestion
- Almost all of the "processors" needed to handle lazer scores are complete, with the exception of play time and medal awarding.
```